packages/shared/forks/ReactFeatureFlags.www.js

```diff
@@ -36,6 +34,10 @@ export const {
 export const enableProfilerTimer = __PROFILE__;
 export const enableProfilerCommitHooks = __PROFILE__;
 
+// TODO: These features do not yet exist in the new reconciler fork.
+export const enableDebugTracing = !__VARIANT__;
+export const enableSchedulingProfiler = !__VARIANT__ && __PROFILE__;
```

Comment:
I think this should just be `__PROFILE__`.

Reply:
Yeah, you're right. I hadn't updated that in this branch yet. My bad.